applications-frontend/src/components/Applicant.vue

```diff
@@ -1,61 +1,83 @@
 <template>
-    <div class="container">
-        <div class="columns">
-            <aside class="review column is-one-quarter">
-                <form>
-                    <div class="field">
-                        <label class="label is-large">Your Score: </label>
-                        <div class="control is-large">
-                            <input class="input" type="number" :min="min" :max="max"/>
-                        </div>
-                    </div>
-                </form>
-            </aside>
-
-            <div class="column is-three-quarters">
-                <!-- random demographic information, can find a nice way to represent or table it -->
-                <section class="basicinfo">
-                    <h1 class="title">John Wick</h1>
-                    <h2 class="subtitle">Baba Yaga</h2>
-                </section>
-                <!-- responses go here -->
-                <section class="responses">
-                
-                </section>
-
-                <!-- random web links can go here -->
-                <section class="links">
-
-                </section>
-            </div>
-        </div>
-    </div>
+<div class="container">
+	<div class="columns">
+
+		<!-- Control panel -->
+		<div class="column has-background-light is-3">		
+			<center>
+				<!-- Person's Image -->
+				<figure class="image is-128x128">
+					<img class="profile-picture" alt="" src="https://bulma.io/images/placeholders/256x256.png">
+				</figure>
+
+				<!-- Person's name -->			
+				<div class="subtitle is-4" style="margin: 16px;">Full-name</div>
+
+				<!-- Break for looking good :D -->
+				<div class="pad-light"></div>
+				<div class="break"></div>
+				<div class="pad-light"></div>
+
+				<!-- Rating control -->
+				<div class="subtitle is-4" style="margin: 16px;">Rating</div>
+				<div class="buttons has-addons is-centered">
```

Comment:
consider using a set of radio buttons and restyling them, this makes getting the value easier. Right now you will need a callback for this set of buttons